docs/sql-ref-syntax-ddl-create-table-hiveformat.md

```diff
@@ -57,9 +63,31 @@ as any order. For example, you can write COMMENT table_comment after TBLPROPERTI
 
     Partitions are created on the table, based on the columns specified.
 
-* **ROW FORMAT**
-
-    SERDE is used to specify a custom SerDe or the DELIMITED clause in order to use the native SerDe.
+* **ROW FORMAT SERDE**
+
+    Specify a custom SerDe.
+    
+* **ROW FORMAT DELIMITED**
+
+    Define DELIMITED clause in order to use the native SerDe.
```

Comment:
`Use the native SerDe to specify the delimiter, escape character, null character, and so on.`

Reply:
https://github.com/apache/spark/blame/master/docs/sql-ref-syntax-dml-insert-overwrite-directory-hive.md#L43